packages/contracts/contracts/core/component/Component.sol

```diff
@@ -6,45 +6,28 @@ pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
-import "@opengsn/contracts/src/BaseRelayRecipient.sol";
 
 import "./Permissions.sol";
 import "../erc165/AdaptiveERC165.sol";
+import "./../acl/ACL.sol";
 import "./../IDAO.sol";
 
-// component => permissions => BaseRelayRecipient, AdaptiveERC165, UUPSUpgradable, Initializable
-
-/// @title The base component in the Aragon DAO framework
+/// @title Base component in the Aragon DAO framework
 /// @author Samuel Furter - Aragon Association - 2021
```

Comment:
should we add co-auther, so the contributions are more visible?

Reply:
We should answer this questions in our development guidelines and change it everywhere (according to the git history).

Reply:
This is noted for another issue, so I will close it here.